nautobot/core/models/fields.py

```diff
@@ -11,8 +11,10 @@
 
 from nautobot.core.constants import CHARFIELD_MAX_LENGTH
 from nautobot.core.forms import fields, widgets
+from nautobot.core.forms.utils import parse_numeric_range
 from nautobot.core.models import ordering
 from nautobot.core.models.managers import TagsManager
+from nautobot.core.models.utils import compress_range
```

Comment:
Wondering if `compress_range` should be implemented in `forms.utils` just to be more self consistent?